docs/source/index.rst

```diff
@@ -0,0 +1,55 @@
+.. andO documentation master file, created by
+   sphinx-quickstart on Tue Mar 30 14:48:19 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to the AnDO project
+===========================
+
+March 2021
+
+The Animal Data Organization (AnDO) is a project initiated at the Institut de Neurosciences de la Timone, in Marseille, France, in order to set up a standardization of the experimental data recorded in animal models. Since its launch in early 2020, its aims have significantly evolve following these axes:
+
+ - first, we have opened our desire for standardization to the community through the launch of a Working Group endorsed by the INCF; please feel free to join the group to participate in this effort: https://www.incf.org/sig/incf-working-group-standardized-data ; we are meeting approximately every two months and we are making progress in subgroups on well defined issues between each meeting;
+
+ - secondly, after discussions within the group, we have initiated a specific standardization effort for electrophysiology, as a BIDS Extension Proposal (BEP32: https://bids.neuroimaging.io/get_involved.html ); we are actively looking for feedback from the community on this proposal, so feel free to directly comment in the document that details the data and metadata organization: http://bit.ly/BIDS-animal-ephys.
+
+At this moment, this repository provides several tools, amongst which a validator (the AnDOChecker) which complies to the latest specifications described in the BEP32 document. We are committed to 
```

Comment:
@Slowblitz I think you forgot to add some text here... the last sentence is incomplete.

Reply:
I think it's my text, ahah ;)

Reply:
@JuliaSprenger This is @SylvainTakerkart 's commit so ...

Reply:
> @JuliaSprenger This is @SylvainTakerkart 's commit so ...

yep, but @Slowblitz , if you see that there's something like this, you should let me know about it ;)